cli/buildPropsData.ts

```diff
@@ -88,8 +88,17 @@ export async function buildPropsData(
   configFile: string,
   verbose: boolean,
 ) {
+  const verboseModeLog = (...messages: any) => {
+    if (verbose) {
+      console.log(...messages)
+    }
+  }
+
+  verboseModeLog('Begging props data build')
```

Comment:
Is this supposed to be beginning?

Reply:
Whoops yeah, this is why I write code not English 😆